.github/workflows/ruby.yml

```
@@ -0,0 +1,35 @@
+name: Ruby Gem
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    types: [ opened, synchronize, reopened, edited, ready_for_review ]
+
+jobs:
+  build:
+    name: Build + Publish
```

Should be just `Build`. 
```suggestion
    name: Build
```